sys/include/net/gnrc/netif.h

```diff
@@ -56,6 +59,16 @@ extern "C" {
  */
 typedef struct gnrc_netif_ops gnrc_netif_ops_t;
 
+#ifdef MODULE_GNRC_NETIF_ASYNC_EVENTS
+/**
+ * @brief   Event type used for passing netdev pointers together with the event
+ */
+typedef struct {
+    event_t super;
+    netdev_t *dev;
+} event_netdev_t;
```

Comment:
Module prefix missing. (I'd rather prefer the name `gnrc_netif_event_t` then ;-))